### ISSUE TITLE
Midterm Report Peer Review - Siyao Gu (sg2238)

### ISSUE THREAD

**Issue by gsymine** [haoqiyao/NASDAQ-Index-Prediction#6]
By: Siyao Gu, sg2238

The group is trying to predict the up/down of NASDAQ 100 index using explanatory variables ranging from risk-free rates, exchange rates to commodity prices, indexes of different markets and Google Trend Index. To solve this classification problem, they make use of a 3618*20 feature matrix. Potential benefits include improved efficiency for professional traders, especially those “lazy” ones with less motivation to look at the screens all the time.

Three things I like about the report:

•	The group obviously has a deep understanding of this topic as they select sufficient and correlated explanatory variables for the prediction model. 10-year treasury bill rate is a commonly used rate for market risk-free rate. Commodity prices measure the global risk. And Google Trend Index represents macroeconomic factors and people’s expectations. Overall, those are reasonable and helpful explanatory variables to use.

•	The group check the distribution of the dependent variable and then approach the problem with a nonlinear model. It is helpful and also interesting to see that the log return of NASDAQ 100 follows a normal distribution. And under a normal distribution assumption, a linear model will tend to make 50-50 up/down prediction based on the 50-50 format of training data it gets and it will have no difference from a random walk. Therefore, starting with a nonlinear model could be a better way to approach the problem.

•	The group has plenty of statistical models to test in the next step. As mentioned in their proposal, they plan to implement Naïve Bayes, Gaussian Discriminant Analysis and SVM besides the Classification Tree and Logistic Regression models they have already used. And in the future development section, they also mention that the following step will be to use the SVM. Hence, there are lots of potentials for the group to enhance the performance of their model.

Three things that may need improvement:

•	As the group has realized, “Shorten the period since data from remote period might cancel out recent data’s significance”. The data used in this project is financial data, which differ from the data that most other groups are using because financial data does discount through time. Specifically, the group is actually digging into a time series classification problem instead of a pure classification problem. That being said, Logistic Regression and even SVM may not be good choices for time series prediction. I further suggestion using ARIMA or Recurrent Neutral Network.

•	The usage of PCA is of doubt. It is not clear from the midterm report that whether the current model is of underfitting problem or overfitting problem. And if the group use PCA to lower the dimensions, they probably already assume that only the underfitting problem exists, which may not be the current case.

•	Logistic Regression may not be used towards the problem. As the group has stated, the data they use follows a nearly normal distribution. Under this distribution, a linear model could probably be trained to make a 50-50 prediction. If we assume the return of NASDAQ 100 index follows a random walk, then a 50-50 prediction made on the random walk could result in a 50% accuracy. As the Logistic Regression actually has a 54.29% accuracy, I would say using this model is not suitable, considering the distribution of the overall data.
